hera_pspec/pspecdata.py

```diff
@@ -864,44 +905,40 @@ def R(self, key):
 
         # Only add to Rkey if a particular mode is enabled
         # If you do add to this, you need to specify this in self.set_R docstring!
-        Rkey = key + (self.data_weighting,) + (self.taper,)
+        Rkey = key + (self.taper,)
```

Comment:
I think `data_weighting` should be part of the key. I think the `set_R` method was missing this element in its doc string description--could you add this element in the doc string of that method?